lib/options.js

```diff
@@ -1,20 +1,21 @@
-const fs = require('fs');
-const path = require('path');
+const { readFileSync, lstatSync } = require('fs');
+const { resolve } = require('path');
 
-const merge = require('lodash/merge');
+const merge = require('merge-options').bind({ concatArrays: true });
+const nodeVersion = require('node-version');
 const weblog = require('webpack-log');
 
 const { load } = require('./config');
+const { apply: applyFlags } = require('./flags');
 
 const defaults = {
   clipboard: true,
   compiler: null,
   config: {},
   content: [],
-  dev: { publicPath: '/' },
-  flags: {},
+  devMiddleware: { publicPath: '/' },
   host: 'localhost',
-  hot: {},
+  // hotClient: {},
```

Comment:
_nit:_ Remove this comment. If you want to keep it, please add a note 😉

Reply:
will do